test/unit/request-auth.test.js

```diff
@@ -438,7 +438,7 @@ describe('RequestAuth', function () {
                 },
                 certificate: undefined,
                 proxy: undefined,
-                body: { },
+                body: { mode: 'raw', raw: { foo: 'bar' } },
```

Comment:
change this test to only look into auth. Why look at entire request?

Reply:
If I recall correctly, this test was failing due to the raw body behaviour change. I'll revisit.